sdk/ml/azure-ai-ml/azure/ai/ml/operations/_data_operations.py

```diff
@@ -544,19 +552,24 @@ def _get_latest_version(self, name: str) -> Data:
         )
         return self.get(name, version=latest_version)
 
-    # pylint: disable=no-self-use
-    def _prepare_to_copy(
```

Comment:
Do we need to add this change to a breaking changes tracker or something similar, since it will break the client's code that uses this method?

Reply:
As we chatted, this is expected for the preview features...